packages/grafana-ui/src/components/Forms/Legacy/Input/Input.tsx

```diff
@@ -4,6 +4,7 @@ import React, { PureComponent, ChangeEvent } from 'react';
 import { ValidationEvents, ValidationRule } from '../../../../types';
 import { validate, EventsWithValidation, hasValidationEvent } from '../../../../utils';
 
+/** @deprecated Please use the non legacy `Input` component, which does not require this enum. */
```

Comment:
What about?

`Please use the `Input` component, which does not require this enum.` 

Better than using negatives.